### ISSUE TITLE
[backstage-plugin-github-insights] Error trying to view plugin widgets

### ISSUE THREAD

**Issue by tatounel** [RoadieHQ/roadie-backstage-plugins#213]
After updating Backstage dependencies, I'm getting the following error on the latest version of the GitHub insights plugin.
```
Error: A <Route> is only ever to be used as the child of <Routes> element, never rendered directly. Please wrap your <Route> in a <Routes>.
    at invariant (webpack-internal:///../../node_modules/@roadiehq/backstage-plugin-github-insights/node_modules/react-router/index.js:52:20)
    at Route (webpack-internal:///../../node_modules/@roadiehq/backstage-plugin-github-insights/node_modules/react-router/index.js:180:12)
    at renderWithHooks (webpack-internal:///../../node_modules/@hot-loader/react-dom/cjs/react-dom.development.js:14804:18)
    at mountIndeterminateComponent (webpack-internal:///../../node_modules/@hot-loader/react-dom/cjs/react-dom.development.js:17483:13)
    at beginWork (webpack-internal:///../../node_modules/@hot-loader/react-dom/cjs/react-dom.development.js:18597:16)
    at HTMLUnknownElement.callCallback (webpack-internal:///../../node_modules/@hot-loader/react-dom/cjs/react-dom.development.js:189:14)
```

**Comment by Irma12** [comment]
Hey @tatounel sorry for the issues you are experiencing. May I ask were you using the plugin before and what version are you upgrading from? Reason I ask is because we weren't able to reproduce it so we need a bit more details :)

**Comment by tatounel** [comment]
Hi @Irma12, 
Thank you for looking into this! Yes, we have been using it for months. I updated our Backstage dependencies and then upgraded the insights plugin from 1.1.22 to 1.1.24 and that's when I noticed the error. Tried to go back to 1.1.22 but still got the error.

**Comment by Irma12** [comment]
Hi @tatounel I have been working on the fix and there is a new release (v1.1.25) so please check it out and let us know if it works :)

**Comment by tatounel** [comment]
Thank you so much @Irma12, it works now!
